packages/terra-arrange/tests/nightwatch/arrange-spec.js

```diff
@@ -101,7 +104,10 @@ module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous
       .url(`${browser.launchUrl}/#/tests/arrange-tests/aligned-individually-fit-start`);
 
     browser.expect.element('#center div:nth-child(1)').to.have.css('align-self').which.equals('center');
-    browser.expect.element('#center div:nth-child(2)').to.have.css('align-self').which.equals('auto');
+    browser.expect.element('#default div:nth-child(2)').to.have.css('align-self').to.not.equal('flex-start');
```

Comment:
Overall we should really avoid checking styling in nightwatch. Since we are not applying a style here via a prop, I'd say we should not be testing this at all.

Reply:
I would rather we tested that the classNames are placed on the element rather than the css itself.